Downstream builds failing due to ambiguous import

Getting a build error due to missing extension specifier in an import:
```
ERROR in ./node_modules/@noble/ciphers/esm/utils.js 2:0-53
Module not found: Error: Can't resolve './_assert' in '/.../node_modules/@noble/ciphers/esm'
Did you mean '_assert.js'?
...
```

Build [logs showing the error](https://github.com/decentralized-identity/did-jwt/actions/runs/7846178209/job/21412261334?pr=307)
Version: `@noble/ciphers@0.5.0`

I believe the fix is simple and will be proposing a PR. See #25

--- a/src/utils.ts
+++ b/src/utils.ts
@@ -1,5 +1,5 @@
 /*! noble-ciphers - MIT License (c) 2023 Paul Miller (paulmillr.com) */
-import { bytes as abytes, isBytes } from './_assert';
+import { bytes as abytes, isBytes } from './_assert.js';
 // prettier-ignore
 export type TypedArray = Int8Array | Uint8ClampedArray | Uint8Array |
   Uint16Array | Int16Array | Uint32Array | Int32Array;

